numpy/core/src/common/simd/avx2/conversion.h

```diff
@@ -26,7 +26,21 @@
 #define npyv_cvt_b32_s32(BL) BL
 #define npyv_cvt_b64_u64(BL) BL
 #define npyv_cvt_b64_s64(BL) BL
-#define npyv_cvt_b32_f32(BL) _mm256_castps_si256(BL)
-#define npyv_cvt_b64_f64(BL) _mm256_castpd_si256(BL)
+#define npyv_cvt_b32_f32 _mm256_castps_si256
+#define npyv_cvt_b64_f64 _mm256_castpd_si256
+
+// convert boolean vector to integer bitfield
+NPY_FINLINE npy_uint64 npyv_tobits_b8(npyv_b8 a)
```

Comment:
The name is a litter confused, how about signmask or movemask?

Reply:
this intrinsic isn't replaceable for `movemask`, since it accepts only boolean vector where the value must be
`(1 << lane_size_inbits) - 1// 0xff****` or `0` and thats why we should get a better perfomance on aarch64.